core/src/html/layout.rs

```diff
@@ -407,6 +407,7 @@ impl<'a, 'gc> LayoutContext<'a, 'gc> {
 
         if let Some(font) = library
             .get_font_by_name(&span.font, span.bold, span.italic)
+            .filter(|f| f.has_glyphs())
```

Comment:
There is content where a font is embedded with a set of glyphs for some text fields, but, mistakenly by the author, other text fields are set to use device fonts and contain non-embedded characters. See #451. Probably it's best to default to Noto Sans always if the text field is set to use device fonts (better to have an incorrect font than missing glyphs?)